cargo-geiger/src/format/print_config.rs

```diff
@@ -22,6 +22,7 @@ pub enum OutputFormat {
     Json,
     GitHubMarkdown,
     Utf8,
+    Ratio,
```

Comment:
Could you order the `OutputFormat` types alphabetically here and in the updated help message please?